pydarn/exceptions/plot_exceptions.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) SuperDARN Canada, Universtiy of Saskatchewan
```

Comment:
University*

Reply:
Is this file new specifically for acf plotting?

Reply:
Oh I see now, it replaced rtp_exceptions

Reply:
Yup!